sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -0,0 +1,347 @@
+# Table of contents
+
+- [Introduction](#introduction)
+- [Authenticating with `DefaultAzureCredential`](#authenticating-with-defaultazurecredential)
+- [Authenticating a user assigned managed identity with `DefaultAzureCredential`](#authenticating-a-user-assigned-managed-identity-with-defaultazurecredential)
+- [(Browsers and NodeJS) Authenticating a user account interactively in the browser](#authenticating-a-user-account-interactively-in-the-browser)
```

Comment:
This helps a bit in getting attention to client side apps...
What do you think of the below?

- Add a h2 for `Authenticating client side browser applications`
   - We can talk about SPA App registration here. https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration and make it clear up front on what such customers would need
- Add a h2 for `Authenticating service side applications`. Then pull all our current h2 to become h3 here

Thoughts?

Reply:
I played around with it some and I'm thinking instead of a table of contents we can have an intro, a section on client side, and a section on server side - in the server side section we can link to the individual credential types from there. That gives you a "start here" and then "choose your own adventure" as you go through the doc. Let me push this up and see what you think